nixos/modules/services/mail/postsrsd.nix

```
@@ -29,9 +50,9 @@ in
         description = "Secret keys used for signing and verification";
       };
 
-      domain = lib.mkOption {
-        type = lib.types.str;
-        description = "Domain name for rewrite";
+      domains = lib.mkOption {
+        type = lib.types.listOf lib.types.str;
+        description = "Domain names for rewrite";
```

This should ideally have a defaultText.

If we're doing that maybe it's worth also moving the default from below (L88) up to here?

```suggestion
        description = "Domain names for rewrite";
        default = [ config.networking.hostName ];
        defaultText = lib.literalExpression "[ config.networking.hostName ]";
```